packages/core/src/cells/uri-cell.tsx

```diff
@@ -115,7 +115,10 @@ export const uriCellRenderer: InternalCellRenderer<UriCell> = {
         const { onChange, value, forceEditMode, validatedSelection } = p;
         return (
             <UriOverlayEditor
-                forceEditMode={forceEditMode || (cell.hoverEffect === true && cell.onClickUri !== undefined)}
+                forceEditMode={
+                    value.readonly !== true &&
```

Comment:
Not sure about this check. It is not really necessary since `readonly` will be anyways enforced within `UriOverlayEditor``